### PR TITLE
feature/203-hover-over-areas

### PR DESCRIPTION
**Pull request by yus252**

### CODE REVIEW

**Comment by yxlin118** [comment]
The overlay of the background map and the map layer is a bit awkward. Maybe use a all black background for the hover?

**Comment by yus252** [comment]
Not only so. You need to redesign it to make it also work on phone. This is just a draft. It still needs front-end design.

**Comment by yus252** [comment]
We will hold on to this feature. We need @ste519 to do the dashboard design.

File: frontend/src/components/Map/index.js [hunk 1]
```diff
@@ -205,10 +207,39 @@ export default function Map(props, { draggable = true }) {
...
+            "</h2><h3>" +
+            country[0].confirmed +
+            " cases confirmed</h3>";
+        } else {
+          document.getElementById("pd").innerHTML =
+            "<h2>" + country_name + "</h2><h3> NA </h3>";
+        }
+      } else {
+        document.getElementById("pd").innerHTML =
+          "<h2> Confirmed Cases </h2> <h3>Hover over a state or country!</h3>";
```
Comment (yxlin118) [inline frontend/src/components/Map/index.js:235]: Would it be better to change it to hover/click for mobile user? Looks really good!

Comment (yus252) [reply frontend/src/components/Map/index.js:235]: Ok

**Pull request merged**